module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+using bitmapArr = std::array<std::array<uint8_t, width>, height>;
+using grayScaleVec = std::vector<std::pair<uint8_t, uint8_t>>;
+
+void printMap(const std::array<std::array<uint8_t, width>, height>& bitmap);
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>&);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(const std::vector<std::pair<uint8_t, uint8_t>>&);
```

Comment:
You have created aliases for longer types. Why haven't you used them here in defenition of your functions?